patches/node/support_v8_sandboxed_pointers.patch

```diff
@@ -24,47 +24,12 @@ index 2abf5994405e8da2a04d1b23b75ccd3658398474..b06e8529bb8ca2fa6d7f0735531bbbf3
 +
  void* NodeArrayBufferAllocator::Allocate(size_t size) {
    void* ret;
--  if (zero_fill_field_ || per_process::cli_options->zero_fill_all_buffers)
--    ret = UncheckedCalloc(size);
-+  if (*zero_fill_field_ || per_process::cli_options->zero_fill_all_buffers)
-+    ret = allocator_->Allocate(size);
-   else
--    ret = UncheckedMalloc(size);
-+    ret = allocator_->AllocateUninitialized(size);
-   if (LIKELY(ret != nullptr))
-     total_mem_usage_.fetch_add(size, std::memory_order_relaxed);
-   return ret;
- }
- 
- void* NodeArrayBufferAllocator::AllocateUninitialized(size_t size) {
--  void* ret = node::UncheckedMalloc(size);
-+  void* ret = allocator_->AllocateUninitialized(size);
-   if (LIKELY(ret != nullptr))
-     total_mem_usage_.fetch_add(size, std::memory_order_relaxed);
-   return ret;
-@@ -100,7 +108,7 @@ void* NodeArrayBufferAllocator::AllocateUninitialized(size_t size) {
- 
- void* NodeArrayBufferAllocator::Reallocate(
-     void* data, size_t old_size, size_t size) {
--  void* ret = UncheckedRealloc<char>(static_cast<char*>(data), size);
-+  void* ret = allocator_->Reallocate(data, old_size, size);
-   if (LIKELY(ret != nullptr) || UNLIKELY(size == 0))
-     total_mem_usage_.fetch_add(size - old_size, std::memory_order_relaxed);
-   return ret;
-@@ -108,7 +116,7 @@ void* NodeArrayBufferAllocator::Reallocate(
- 
- void NodeArrayBufferAllocator::Free(void* data, size_t size) {
-   total_mem_usage_.fetch_sub(size, std::memory_order_relaxed);
--  free(data);
-+  allocator_->Free(data, size);
- }
- 
- DebuggingArrayBufferAllocator::~DebuggingArrayBufferAllocator() {
+   if (zero_fill_field_ || per_process::cli_options->zero_fill_all_buffers)
```

Comment:
Does the deletion here indicate part of the patch got upstreamed? Or, is it by luck that upstream also changed those lines?

Reply:
this part of the patch got upstreamed in https://github.com/nodejs/node/pull/43594